### PR TITLE
Fix invalid name issue with matlab structs.

### PR DESCRIPTION
**Pull request by chrisdembia**
### Brief summary of changes

The syntax for input connectee names changed, requiring an update for how struct
fieldnames are formed.

### Testing I've completed

Locally ran `ctest -R Matlab_osimTableToStruct -V`.

### CHANGELOG.md (choose one)

- no need to update because...this file is new to 4.0.

### CODE REVIEW

File: Bindings/Java/Matlab/Utilities/osimTableToStruct.m [hunk 1]
```diff
@@ -77,9 +77,15 @@
...
     col_label  = char(osimtable.getColumnLabels.get(iLabel));
-    % replace fwd slashes if present
-    if isempty(strfind(col_label, '/'))
+    
+    % MATLAB structs must start with a letter, and can only contain
+    % letters, digits, and underscores.
+    % Remove an initial slash.
+    col_label = regexprep(col_label, '^/', '');
+    % replace '/' and '|' if present
+    if ~isempty(strfind(col_label, '/')) || ~isempty(strfind(col_label, '|'))
```
Comment (tkuchida) [inline Bindings/Java/Matlab/Utilities/osimTableToStruct.m]: (Unnecessary to perform this check.)

Comment (chrisdembia) [inline Bindings/Java/Matlab/Utilities/osimTableToStruct.m]: Addressed.

**Comment by chrisdembia** [comment]
I killed the PRs because the MATLAB tests do not run on travis/appveyor (I forgot to use `[skip ci]`).

**Comment by aseth1** [comment]
LGTM

**Pull request merged**